tools/ClassGen/NodeGen.cpp

```diff
@@ -1091,6 +1091,15 @@ int main(int argc, char **argv) {
       .setDocstring(
           "Applies ReLU, max(0, x), to each element in the Input tensor.");
 
+  BB.newNode("HardSigmoid")
+      .addInput("Input")
+      .addMember(MemberType::Float, "Alpha")
+      .addMember(MemberType::Float, "Beta")
+      .addResultFromCtorArg()
+      .dataParallel()
+      .setDocstring("Applies HardSigmoid, max(0, min(1, alpha * x + beta)) "
+                    "to each element in the Input tensor.");
+
```

Comment:
We don't need this anymore!

Reply:
Oops..
done
4f1486f6cf04cbeca26a0ea40a631307ea0e0448